app/helpers/textual_mixins/textual_refresh_status.rb

```diff
@@ -1,20 +1,32 @@
 module TextualMixins::TextualRefreshStatus
+  include ActionView::Helpers::DateHelper
   def textual_refresh_status
-    last_refresh_status = @record.last_refresh_status.titleize
-    if @record.last_refresh_date
-      last_refresh_date = time_ago_in_words(@record.last_refresh_date.in_time_zone(Time.zone)).titleize
+    refresh_status(@record)
+  end
+
+  def textual_refresh_date
+    refresh_date(@record)
+  end
+
+  def refresh_status(record)
+    last_refresh_status = record.last_refresh_status&.titleize
+    refresh_time = record.last_refresh_date&.in_time_zone(Time.zone)
```

Comment:
looking at this, I'm not sure it's right, or that it needs `Time.zone`.  We aren't going to present the time in any particular time zone (if we did, I'd say we should use the user's time zone), but instead we are using relative time, which doesn't need a time zone.  I think we should remove it.

Reply:
Ok let me try removing that and see